lua/terrortown/menus/score/info.lua

```diff
@@ -29,11 +29,16 @@ local function MakePlayerRoleTooltip(parent, width, ply)
 		plyRoleBox.GetTitle = function()
 			return tostring(i) .. ". " .. TryT(roleData.name) .. " (" .. TryT(plyRole.team) .. ")"
 		end
-
+		local length = fastutf8.len(plyRoleBox.GetTitle())
+		if length > shortest then
+			shortest = length
+		end
 		height = height + 20
 	end
 
-	return height
+	width = (shortest * 8)
```

Comment:
brackets are not needed here

Reply:
also where is the magic 8 coming from?

Reply:
This happens three more times, maybe even put it in a function?

Reply:
getting the font for finding the string length is too far out since the paint function controls its presentation and not dlabel stuff, so it's ballparking it

the three functions are three functions, i don't know how much i can really combine them though

Reply:
actually, found a way that i might be able to deduplicate and work toward the stated goal of unraveling things at the same time